openlibrary/data/__init__.py

```diff
@@ -9,7 +9,7 @@ def parse_data_table(filename):
     <key, type, revision, json> for all entries.
     """
     for line in open(filename):
-        thing_id, revision, json = pgdecode(line).strip().split("\t")
-        d = simplejson.loads(json)
+        thing_id, revision, json_data = pgdecode(line).strip().split("\t")
+        d = json.loads(json_data)
         yield d['key'], d['type']['key'], str(d['revision']), json
```

Comment:
BLOCKER: I think this is wrong -- this line now references `json` the library, not `json_data`

Reply:
Good catch!